src/terrain.ts

```diff
@@ -112,20 +130,26 @@ type Options = {
  *   attribution: '<a href="https://gbank.gsj.jp/seamless/elev/">産総研シームレス標高タイル</a>',
  * });
  */
-export const useGsiTerrainSource = (addProtocol: typeof maplibregl.addProtocol, options: Options = {}): RasterDEMSourceSpecification => {
-    addProtocol('gsidem', (params, abortController) => {
-        const urlWithoutProtocol = params.url.replace('gsidem://', '');
-        const url = new URL(urlWithoutProtocol);
-        return workerProtocol.request(url, abortController);
-    });
-    const tileUrl = options.tileUrl ?? `https://cyberjapandata.gsi.go.jp/xyz/dem_png/{z}/{x}/{y}.png`;
+export const useGsiTerrainSource = (
+	addProtocol: typeof maplibregl.addProtocol,
+	options: Options = {},
+): RasterDEMSourceSpecification => {
+	addProtocol('gsidem', (params, abortController) => {
+		const urlWithoutProtocol = params.url.replace('gsidem://', '');
+		return workerProtocol.request(urlWithoutProtocol, abortController);
+	});
+	const tileUrl =
+		options.tileUrl ??
+		`https://cyberjapandata.gsi.go.jp/xyz/dem_png/{z}/{x}/{y}.png`;
 
-    return {
-        type: 'raster-dem',
-        tiles: [`gsidem://${tileUrl}?x={x}&y={y}&z={z}`],
-        tileSize: 256,
-        minzoom: options.minzoom ?? 1,
-        maxzoom: options.maxzoom ?? 14,
-        attribution: options.attribution ?? '<a href="https://maps.gsi.go.jp/development/ichiran.html">地理院タイル</a>',
-    };
+	return {
+		type: 'raster-dem',
+		tiles: [`gsidem://${tileUrl}?x={x}&y={y}&z={z}`],
```

Comment:
xyzのURLパラメーターは使用しなくなったので削除でよい？
```suggestion
		tiles: [`gsidem://${tileUrl}`],
```